docker-compose.yml

```diff
@@ -447,10 +447,9 @@ services:
     container_name: smtp-server
     image: gessnerfl/fake-smtp-server
     environment:
-      - "SERVER-PORT=8080"
```

Comment:
This variable is not actually used.

Reply:
The problem is the variable is spelled wrong, it should be `SERVER_PORT`. Dashes in environment variables are never supported. See https://github.com/gessnerfl/fake-smtp-server#configuration

Reply:
Fixed.